proto/zitadel/feature/v2/instance.proto

```diff
@@ -93,6 +93,13 @@ message SetInstanceFeaturesRequest{
       description: "If the flag is enabled, you'll be able to use the OIDC Back-Channel Logout to be notified in your application about terminated user sessions.";
     }
   ];
+
+  optional LoginV2 login_v2 = 13 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "true";
+      description: "If the flag is set, all users will be redirected to the login V2 regardless of the application's preference.";
```

Comment:
I would say the description of this message is a bit different, something along the lines of 'If the flag is set, it impacts the usage of the login v2 regardless of the application's preferences.?